cli/cmd/install.go

```diff
@@ -511,7 +537,19 @@ func (options *installOptions) globalConfig(identity *pb.IdentityContext) *pb.Gl
 		CniEnabled:        options.noInitContainer,
 		Version:           options.linkerdVersion,
 		IdentityContext:   identity,
-		InstallationUuid:  id,
+	}
+}
+
+func (options *installOptions) installConfig() *pb.Install {
+	installID := uuid.NewV4().String()
```

Comment:
TIOLI: Depending on how you feel about globals... 
```
// install.go
var generateID = func() string {
  return uuid.NewV4().String()
}

func (options *installOptions) installConfig() *pb.Install {
	installID := generateID()
    ...
}

// install_test.go
generateID = func() string {
  return "deaab91a-f4ab-448a-b7d1-c832a2fa0a60"
}
```
(Personally, I'm ok using it here since it's a short-lived CLI process :stuck_out_tongue:)

Reply:
How do you feel about hanging this off of `installOptions` instead of as a global? The `cmd` package is prettttty big, so i'd rather keep this scoped somehow.